### ISSUE TITLE
[enhancement] Enable search to find `br` and `br_if`

### ISSUE THREAD

**Issue by 99991** [WebAssembly/spec#1097]
tl;dr
https://webassembly.github.io/spec/core/search.html?q=br&check_keywords=yes&area=default
does not find `br` and only one page with `br_if`.

I have spent more time than I'd like to admit on figuring out how blocks and loops work. I'll describe my troubles here in case other people are having the same problem.

`br` is an unfortunate name, because people with a background in C++ (like me) might think that it stands for `break` when it is actually `branch`. The following wasm code
```
(loop $label
    (br $label)
)
```
might then incorrectly be interpreted to be equivalent to the following C++ code
```
while (true){
    break;
}
```
when it is actually
```
label:{
    goto label;
}
```
i.e. the `br` jumps to the beginning of the loop and the `loop` instruction does not actually loop on its own.

The fact that `br` changes jump direction depending on if it is enclosed in either a `block` or a `loop` did not help either.

Of course I've only got myself to blame since I should have read the spec ( https://webassembly.github.io/reference-types/core/syntax/instructions.html#control-instructions ) beforehand instead of just random guessing, but even if I tried, I would not have found it.

**Comment by rossberg** [comment]
There is an index listing all instructions: http://webassembly.github.io/spec/core/appendix/index-instructions.html

However, the spec is not intended as a Wasm tutorial. For that, there is better material, e.g. [MDN](https://developer.mozilla.org/en-US/docs/WebAssembly).

As for `br`, you can view it as a break even for a loop. The `br` in
```
(loop $l ... (br $l) ...)
```
corresponds to the `break` in the following JavaScript:
```
while (true) l: { ... break l; ...}
```

**Comment by binji** [comment]
It seems the question has been answered here, closing.
